nextclade-cli: Unspecific error message when missing parameter --input-qc-config

Hi,

when I run nextclade v 1.4.1 (from bioconda) with
```
nextclade --input-fasta myinput.fa --input-root-seq MN908947.3.fasta --input-tree data/sars-cov-2/tree.json --input-gene-map data/sars-cov-2/genemap.gff --output-tsv output/nextclade.tsv --output-dir output
```
it prints the error
```
[ERROR] Nextclade: Error: File not found: "". Please verify correctness of input flags. If a dataset is used, check that the dataset is not corrupted.
```

Adding option `--input-qc-config data/sars-cov-2/qc.json` solves the problem and nextclade runs through.

--- a/packages/nextclade_cli/src/io/getInputPaths.h
+++ b/packages/nextclade_cli/src/io/getInputPaths.h
@@ -89,6 +89,11 @@ namespace Nextclade {
         "Either `--input-dataset` or `--input-tree` is required. Cannot proceed without reference tree.");
     }
 
+    if (cliParams->inputDataset.empty() && inputQcConfig.empty()) {
+      throw ErrorFatal(
+        "Either `--input-dataset` or `--input-qc-config` is required. Cannot proceed without QC config.");
+    }
+
     if (!cliParams->inputDataset.empty()) {
       auto inputDataset = std::string{cliParams->inputDataset};
 

